src/native/Net.h

```diff
@@ -230,7 +237,7 @@ class NetworkManager : public StaticSharedEbb<NetworkManager> {
     explicit Interface(EthernetDevice& ether_dev)
         : address_(nullptr), ether_dev_(ether_dev) {}
 
-    void Receive(std::unique_ptr<MutIOBuf> buf);
+    void Receive(std::unique_ptr<MutIOBuf> buf, uint64_t rxflag);
```

Comment:
If you set the default value for rxflag = 0 you can revert the changes made in VirtIONet.cc

Reply:
agreed